src/Driver/SolverTypes/MISSolverType.jl

```diff
@@ -148,21 +170,66 @@ function solversetup(
     # slower processes (advection and diffusion)
     if ode_solver.discrete_splitting
         numerical_flux_first_order =
-            (dg.numerical_flux_first_order, (dg.numerical_flux_first_order,))
+            (ode_solver.numerical_flux_first_order_slow, (ode_solver.numerical_flux_first_order_slow,))
```

Comment:
I see. So you want to distinguish what fluxes you want to use for the different parts. OK, we should talk with @simonbyrne about some ways we can do this without having to pass so many arguments